test/ttmlir/Silicon/TTNN/pooling/simple_pooling.mlir

```diff
@@ -1,12 +1,14 @@
-// RUN: ttmlir-opt --ttir-to-ttnn-backend-pipeline="system-desc-path=%system_desc_path%" %s > %t.mlir
-// RUN: FileCheck %s --input-file=%t.mlir
-// RUN: ttmlir-translate --ttnn-to-flatbuffer %t.mlir > %t.ttnn
+// RUN: ttmlir-opt --ttir-to-ttnn-backend-pipeline %s | FileCheck %s
```

Comment:
Shouldn't silicon test run the whole pipeline?

Reply:
Yep good catch